helloWorld.js

```diff
@@ -0,0 +1,5 @@
+function greeting(parameterVariable) {
```

Comment:
what is the argument type

Reply:
It can be anything are just printing the provided parameter

Reply:
if the argument is undefined, will you still print the undefined  ?